storybook/pages/TransactionDelegatePage.qml

```diff
@@ -0,0 +1,180 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import Storybook 1.0
+import utils 1.0
+
+import shared.controls 1.0
+
+SplitView {
+    id: root
+
+    property QtObject mockupModelData: QtObject {
```

Comment:
are you no using any of these properties?
if so remove them

Reply:
Fixed